cli/config/configfile/file.go

```diff
@@ -43,6 +43,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	InsecureRegistries   []string                    `json:"insecure-registries,omitempty"`
```

Comment:
I'm not sure about this.

I guess we currently store the list of insecure registries in the daemon config. With this change we'd have to duplicate the list in the client config.

We already query the engine for the default index server, so we might need to do that here as well.